package/boot/uboot-layerscape-armv7/patches/001-increse-CONFIG_SYS_BOOTM_LEN.patch

```diff
@@ -0,0 +1,26 @@
+From c268f9cdc3cb7bceb323a77a7f857d546a5bd30d Mon Sep 17 00:00:00 2001
+From: Xiaoliang Yang <xiaoliang.yang@nxp.com>
+Date: Tue, 11 Apr 2017 18:03:14 +0800
+Subject: [PATCH] increse CONFIG_SYS_BOOTM_LEN to 10M
```

Comment:
I don't really feel this is any more descriptive than the original comment :)

Reply:
The kernel image will larger than 8M Byte after I add some features on kernel configuration, so I increase the CONFIG_SYS_BOOTM_LEN in u-boot. Do you mean that I need to describe the reason why to increase the length?